.github/workflows/dart_ci.yml

```diff
@@ -0,0 +1,55 @@
+name: Dart CI
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'test_consume_*'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [ 2.13.4, stable, beta, dev ]
```

Comment:
Pretty sure this was 2.7.2 in the travis.yml but that caused issues here. Is this change okay? Should it just be `[ 2.13.4 ]`?

Reply:
This looks good to me. We don't need to test on 2.7.2 anymore.

If anything, we could remove the `beta` channel since we already have the `dev` channel, but 🤷 .

Reply:
I prefer having both `beta` and `dev` because they dev is more likely to break. If we have both, we can usually ignore the dev failures and even report them in hopes that they get fixed before beta.